app/views/layouts/application.html.erb

```diff
@@ -40,46 +40,65 @@
+              ></path>
+
+              <image src="<%=image_url('govuk-logotype-crown.png') %>" class="govuk-header__logotype-crown-fallback-image"></image>
+            </svg>
+            <span class="govuk-header__logotype-text">
+              GOV.UK
             </span>
-        </a>
-      </div>
-      <div class="govuk-header__content">
-        <a href="#" class="govuk-header__link govuk-header__link--service-name">
-          GovWifi
-        </a>
-        <% if user_signed_in? %>
-            <%= link_to 'Logout', destroy_user_session_path, method: :delete, class: "govuk-header__link" %>
-          <% else %>
-            <%= link_to 'Sign Up', new_user_registration_path, class: "govuk-header__link" %> |
-            <%= link_to 'Login', new_user_session_path, class: "govuk-header__link" %>
+          </span>
         <% end %>
       </div>
+
+      <div class="govuk-header__content d-flex justify-content-between">
+        <%= link_to "GovWifi", root_path, class: "govuk-header__link govuk-header__link--service-name" %>
+        <button role="button" class="govuk-header__menu-button js-header-toggle" aria-controls="navigation" aria-label="Show or hide Top Level Navigation">Menu</button>
+        <nav>
+          <ul id="navigation" class="govuk-header__navigation " aria-label="Top Level Navigation">
+            <% if user_signed_in? %>
+              <li class="govuk-header__navigation-item">
+                <%= link_to 'Logout', destroy_user_session_path, method: :delete, class: "govuk-header__link" %>
+              </li>
+            <% else %>
+              <li class="govuk-header__navigation-item">
+                <%= link_to 'Sign Up', new_user_registration_path, class: "govuk-header__link govuk-!-margin-right-3" %>
+              </li>
+              <li class="govuk-header__navigation-item">
+                <%= link_to 'Login', new_user_session_path, class: "govuk-header__link" %>
+              </li>
+            <% end %>
+          </ul>
+        </nav>
+      </div>
     </div>
   </header>
 
   <div class="govuk-width-container">
+    <div class="govuk-phase-banner">
+      <p class="govuk-phase-banner__content">
+        <strong class="govuk-tag govuk-phase-banner__content__tag ">alpha</strong>
+        <span class="govuk-phase-banner__text">
+          This is a new service – your
+          <%= mail_to "support@govuk.zendesk.com", "feedback", class: "govuk-link" %> will help us to improve it.
```

Comment:
Do we want to send feedback to that address? Or should we use our govwifi-specific address?